src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -212,8 +212,17 @@ public sealed partial class SqlBulkCopy : System.IDisposable
         public void Close() { }
         void System.IDisposable.Dispose() { }
         public void WriteToServer(System.Data.Common.DbDataReader reader) { }
+        public void WriteToServer(System.Data.DataTable table) { }
```

Comment:
These new APIs will need a version update to SqlClient ref assembly. 
I am not sure how that mechanism is handled now. The versions used to reside in the csproj file. 

@weshaggard What is the guidance here?